src/components/common.js

```diff
@@ -33,20 +33,20 @@ export const Clickable = ({ as = 'div', disabled, tooltip, tooltipSide, onClick,
   }
 }
 
-const linkProps = disabled => ({
+const linkProps = ({ disabled, variant }) => ({
   as: 'a',
   style: {
-    color: disabled ? colors.gray[2] : colors.green[0],
+    color: disabled ? colors.gray[2] : variant === 'light' ? colors.lightGreen[0] : colors.green[0],
     cursor: disabled ? 'not-allowed' : 'pointer',
     fontWeight: 500
   },
-  hover: disabled ? undefined : { color: colors.green[1] }
+  hover: disabled ? undefined : { color: variant === 'light' ? colors.lightGreen[1] : colors.green[1] }
 })
 
-export const link = ({ onClick, href, ...props }, children) => {
+export const link = ({ onClick, href, disabled, variant, ...props }, children) => {
   return h(Interactive,
-    _.merge(linkProps(props.disabled), {
-      href,
+    _.merge(linkProps({ disabled, variant }), {
+      href, disabled,
```

Comment:
We discovered that `disabled` isn't valid on an `a` or used by `Interactive`.

Reply:
I'm going to tackle this in a separate roundup of common component work.